### PR TITLE
feat(i18n): migrate notification page translation

### PR DESCRIPTION
**Pull request by Jasmine-liang**

### CODE REVIEW

File: src/components/Notice/ArticleNotice/ArticleNewAppreciationNotice.tsx [hunk 1]
```diff
@@ -40,11 +40,9 @@ const ArticleNewAppreciationNotice = ({
       <section className="content-wrap">
         <NoticeHead>
           <NoticeHeadActors actors={notice.actors} />
-
-          <Translate
-            zh_hant="喜歡並讚賞了你的作品"
-            zh_hans="喜欢并赞赏了你的作品"
-            en=" liked your article"
+          <FormattedMessage
+            defaultMessage="liked your article"
```
Comment (robertu7) [inline src/components/Notice/ArticleNotice/ArticleNewAppreciationNotice.tsx:44]: ` liked your article`?

File: src/components/Notice/ArticleNotice/ArticleNewSubscriberNotice.tsx [hunk 2]
```diff
@@ -46,18 +46,19 @@ const ArticleNewSubscriberNotice = ({
               <NoticeActorName user={actor} />
               {isMultiActors && index < 1 && <span>、</span>}
             </Fragment>
-          ))}{' '}
```
Comment (robertu7) [inline src/components/Notice/ArticleNotice/ArticleNewSubscriberNotice.tsx:49]: sure to remove the whitespace?

File: src/components/Notice/ArticleNotice/CircleNewArticle.tsx [hunk 3]
```diff
@@ -31,16 +31,15 @@ const CircleNewArticle = ({
...
           subtitle={
-            <Translate
-              zh_hant="新作品已經加入圍爐，馬上前往閱讀吧！"
-              zh_hans="新作品已经加入围炉，马上前往阅读吧！"
-              en="A new article has been added to the circle, read it now!"
+            <FormattedMessage
+              description="src/components/Notice/ArticleNotice/CircleNewArticle.tsx"
+              defaultMessage="A new article has been added to the circle, read it now! {circle} is growing"
+              values={{
+                circle: <NoticeCircleName circle={circle} />,
```
Comment (robertu7) [inline src/components/Notice/ArticleNotice/CircleNewArticle.tsx:38]: Is it work for passing a React component?

File: src/components/Notice/CircleNotice/CircleInvitationNotice.tsx [hunk 4]
```diff
@@ -31,18 +31,19 @@ const CircleInvitationNotice = ({
...
+            defaultMessage="Congrats!"
+            description="src/components/Notice/CircleNotice/CircleInvitationNotice.tsx"
+          />
           <NoticeActorName user={actor} />
-          <Translate
-            zh_hant=" 邀請你免費加入圍爐 "
-            zh_hans=" 邀请你免费加入围炉 "
-            en=" invites you to join Circle for "
+          <FormattedMessage
+            defaultMessage="invites you to join Circle for"
```
Comment (robertu7) [inline src/components/Notice/CircleNotice/CircleInvitationNotice.tsx:40]: missing whitespace?

File: src/components/Notice/CircleNotice/CircleNewBroadcastComments.tsx [hunk 5]
```diff
@@ -70,38 +71,52 @@ const CircleNewBroadcastComments = ({
...
+                  circlename: (
+                    <NoticeCircleName
+                      circle={notice.circle}
+                      path={circleCommentPath}
+                    />
+                  ),
+                }}
               />
             )}
+            <FormattedMessage
```
Comment (robertu7) [inline src/components/Notice/CircleNotice/CircleNewBroadcastComments.tsx:98]: Is it duplicated with L79-L81?

File: src/components/Notice/CircleNotice/CircleNewDiscussionComments.tsx [hunk 6]
```diff
@@ -80,13 +81,16 @@ const CircleNewDiscussionComments = ({
             {!isEn ? (
               <>
                 {isCircleOwner ? (
-                  <Translate
-                    zh_hant="在你的圍爐 "
-                    zh_hans="在你的围炉 "
-                    en=" in your circle "
+                  <FormattedMessage
+                    defaultMessage="commented in your circle "
```
Comment (robertu7) [inline src/components/Notice/CircleNotice/CircleNewDiscussionComments.tsx:85]: ` in your circle`?

File: lang/zh-Hans.json [hunk 7]
```diff
@@ -42,5 +42,213 @@
...
+  "KZlRde": {
+    "defaultMessage": " 关联了你的作品 ",
+    "description": "src/components/Notice/ArticleArticleNotice/ArticleNewCollectedNotice.tsx"
+  },
+  "/nu5ho": {
+    "defaultMessage": "回复了你在作品 ",
+    "description": "src/components/Notice/CommentCommentNotice/CommentNewReplyNotice.tsx"
+  },
+  "ZUPQzl": {
+    "defaultMessage": "的评论",
```
Comment (robertu7) [inline lang/zh-Hans.json:59]: missing whitespace?

File: src/components/Notice/ArticleNotice/ArticleNewSubscriberNotice.tsx (hunk 2 above)
Comment (Jasmine-liang) [reply src/components/Notice/ArticleNotice/ArticleNewSubscriberNotice.tsx:49]: Added back👌

**Review by robertu7 — APPROVED** [review]

**Pull request merged**